zebrad/src/components/mempool/error.rs

```diff
@@ -43,4 +43,8 @@ pub enum MempoolError {
     /// The queue's capacity is [`super::downloads::MAX_INBOUND_CONCURRENCY`].
     #[error("transaction dropped because the queue is full")]
     FullQueue,
+
+    /// The transaction conflicts with another transaction already in the mempool.
+    #[error("transaction rejected because it conflicts with another transaction in the mempool")]
+    Conflict,
```

Comment:
Name/Log Nitpick: It might help to identify this as a spend conflict, because there are multiple kinds of conflicts.

For example, `InMempool`, `InBlock`, and `AlreadyQueued` are all conflicts.

```suggestion
    /// The transaction has a spend conflict with another transaction already in the mempool.
    #[error("transaction rejected because another transaction in the mempool has already spent some of its inputs")]
    SpendConflict,
```